src/nvim/globals.h

```diff
@@ -140,7 +140,7 @@ EXTERN int mod_mask INIT(= 0x0);                /* current key modifiers */
  * temporarily moved.  The old position is restored with the next call to
  * update_screen().
  */
-EXTERN int cmdline_row;
+EXTERN long cmdline_row;
```

Comment:
> In what cases?

My wording was poor, I meant `in this case`. I was (possibly incorrectly) conflating `assert` with something like `memset_s`. I'm not particularly experienced with C, so these may seem like silly questions.